apps/aecore/src/aec_conductor.erl

```diff
@@ -812,15 +838,31 @@ start_mining_(#state{key_block_candidates = [{HeaderBin, Candidate} | Candidates
             State3 = register_miner_instance(Instance, Pid, State2),
             epoch_mining:info("Miner ~p started", [Pid]),
             start_mining_(State3)
-    end.
+    end;
+start_mining_(#state{stratum_mode = true,
+                     key_block_candidates = [{HeaderBin, #candidate{refs = StratumRefs} = Candidate} | Candidates]} = State)
+  when StratumRefs =:= 0  ->
+    epoch_mining:info("Stratum dispatch ~p", [HeaderBin]),
+    Target            = aec_blocks:target(Candidate#candidate.block),
+    Info              = [{top_block_hash, State#state.top_block_hash}],
+    Server            = self(),
+    aec_events:publish(start_mining, Info), %% TODO: check if need to swap for stratum specific
```

Comment:
I did not finish analyzing conductor, though we shall address the TODOs.

Reply:
Leftover, removed.